.travis.yml

```diff
@@ -193,17 +195,21 @@ matrix:
     - env:
         - DESC="spell checker"
         - CMD="./.ci/test-spelling-unknown-words.sh"
+        - FORCE_CI = true
 
 script:
   - SKIP_FILES=".github|codeship-*|buddy.yml|appveyor.yml|circleci|distelli-manifest.yml|fast-forward-merge.sh|LICENSE|LICENSE.apache20|README.md|release.sh|RIGHTS.antlr|shippable.yml|shippable.sh|wercker.yml|wercker.sh|intellij-idea-inspections.xml"
-  - SKIP_CI=$(if [[ $(git diff --name-only HEAD HEAD~1 | grep -vE "$SKIP_FILES" | cat | wc -c | sed 's/^ *//' ) > 0 ]]; then echo false; else echo true; fi;)
+  - SKIP_CI=$(if [[ $(git diff --name-only HEAD HEAD~1 | grep -vE "$SKIP_FILES" | cat | wc -c | sed 's/^ *//' ) > 0 && -z ${FORCE_CI+x} ]]; then echo false; else echo true; fi;)
```

Comment:
lets "rename" SKIP_CI to SKIP_BY_FILES.

Reply:
do we want to keep the rename if we keep the `FORCE_CI` in here?